common/check_fts_backlog

```diff
@@ -190,10 +179,10 @@ if __name__ == "__main__":
     except:
         sys.exit(WORST_RETVALUE)
 
+    # Does this not do the same thing as the query? Why the duplicate? 
```

Comment:
Ah, this is a portion of code which ATLAS uses to actually set distances. CMS does not do this here (we have another script to do it).

Reply:
We leave the script at line 156